lib/active_model_serializers/adapter/hypertext_application_language/link.rb

```diff
@@ -0,0 +1,77 @@
+module ActiveModelSerializers
+  module Adapter
+    class HypertextApplicationLanguage
+      class Link
+        include SerializationContext::UrlHelpers
+
+        def initialize(serializer, value)
+          # handles warning actionpack-4.0.13/lib/action_dispatch/routing/route_set.rb
+          # warning: instance variable @_routes not initialized
+          @_routes ||= nil
```

Comment:
I thought I fixed that at some point... owait, maybe you based this on that? https://github.com/rails-api/active_model_serializers/pull/1624/files